tools/explorer/explorer.py

```diff
@@ -11,54 +11,46 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Main module for DBO explorer."""
+import pyfiglet
 import sys
 
+from lib import explorer_handler
+from lib import parse_input
```

Comment:
where is this file? I can't find it ? :)

Reply:
Yikes. I forgot to commit the new file.....my bad